tools/clang/lib/SPIRV/ModuleBuilder.cpp

```diff
@@ -80,26 +82,40 @@ uint32_t ModuleBuilder::bbCreate() {
   return labelId;
 }
 
-bool ModuleBuilder::bbReturn(uint32_t labelId) {
+bool ModuleBuilder::setInsertPoint(uint32_t labelId) {
   auto it = basicBlocks.find(labelId);
   if (it == basicBlocks.end()) {
```

Comment:
Hmm, it depends how we define the API in `ModuleBuilder`. There are two approaches currently in `ModuleBuilder`:

* Return on valid execution path and explode on invalid usage (violating assumptions). Various instruction insertion methods are following this.
* Return valid ids on valid execution path and return 0 on invalid usage. Function/BasicBlock creating/ending methods are following this.

I kinda think `setInsertPoint` belongs to the second case.

But it also makes sense to unify these two approaches.